### ISSUE TITLE
 signOut unnecessary complicated?

### ISSUE THREAD

**Issue by George713** [j4w8n/sveltekit-supabase-ssr#5]
The logout-event is triggered on the frontend from `+layout.svelte` via a post request. However, wouldn't it be simpler to just call
```js
supabase.auth.signOut():
invalidate();
```
?
I wonder what the benefits of the other implementation are. Is it more secure or does it simply allow for additional business logic on the backend when a logout-event is triggered?

___
PS: Love the template! Learned a lot from it!

**Comment by j4w8n** [comment]
Yeah, you can do that. You'll likely wanna determine any post-signout redirects - back to / or /auth for example.

I built this strictly for server side auth, so I wanted to remain consistent with the sign out process. Plus I also wanted to try using httponly cookies (available in another branch) which definitely requires server side sign out.
